packages/extra/src/Querying/ConditionParsers/Drupal/DrupalConditionParser.php

```diff
@@ -51,8 +51,14 @@ public function parseCondition($condition): PathsBasedInterface
             return $pathSegment;
         }, explode('.', $pathString));
 
-        return array_key_exists(DrupalFilterParser::VALUE, $condition)
-            ? $this->drupalConditionFactory->createConditionWithValue($operatorName, $condition[DrupalFilterParser::VALUE], $path)
-            : $this->drupalConditionFactory->createConditionWithoutValue($operatorName, $path);
+        if (array_key_exists(DrupalFilterParser::VALUE, $condition )) {
+            if (!array_key_exists(DrupalFilterParser::OPERATOR, $condition )
+                || ($condition[DrupalFilterParser::OPERATOR] !== 'IS NULL'
+                        && $condition[DrupalFilterParser::OPERATOR] !== 'IS NOT NULL')) {
+                return $this->drupalConditionFactory->createConditionWithValue($operatorName, $condition[DrupalFilterParser::VALUE], $path);
+            }
+        }
```

Comment:
just some information about why this came up... 
![image](https://github.com/user-attachments/assets/f0d3af20-3fc5-47ea-b542-971bb502affb)
We can fix this issue outside of EDT. This was still a BC introduced by splitting this method so maybee it is not intended.

Reply:
How do you intend to handle the value `unassigned` in this example?

Reply:
There is no need for this value 'unassigned' appart from informing the FE about there filterChoice and we tried removing it. 
If the value is not set - we get a constraint violation within the drupalFilterValidator that complains about a missing Value. But that means the condition to call the method createConditionWithoutValue cant be met.

Reply:
> If the value is not set - we get a constraint violation within the drupalFilterValidator 

@MoritzMandler: In that case I'd like to verify the correct behavior of `DrupalFilterValidator`. From what you wrote I deduced the following test case. It succeeds in my environment, though I'm not sure regarding the exact branch you're on. Could you please adjust it based on your actual observations, so that the mentioned constraint violation is thrown?
```php
$factory = new PredefinedDrupalConditionFactory(
    new DqlConditionFactory()
);
$validator = new DrupalFilterValidator(
    Validation::createValidator(),
    $factory
);
$validator->validateFilter([
    "foo" => [
        "condition" => [
            "path" => "assignee",
            "operator" => "IS NULL"
        ]
    ]
]);
```

Reply:
Thanks a lot for your help.
We found out that the request was falsely structured after removing the value 'unassigned'.
By fixing it everything works fine without the need to change the EDT stock behavior.
Here was the wrong format that gave us these headaches...
![image](https://github.com/user-attachments/assets/19ec50fe-9e53-4ef7-8608-4a97f9ff17e4)